File: queries/_test_fixtures/javascript/simple.js
/**
 * Simple JavaScript module for AST fixture testing.
 */

class SimpleService {
    constructor(name) {
        this.name = name;
    }

    getName() {
        return this.name;
    }

    setName(name) {
        this.name = name;
    }
}

function standaloneFunction(x) {
    return x * 2;
}

module.exports = { SimpleService, standaloneFunction };

File: queries/_test_fixtures/python/simple.py
"""Simple Python module for AST fixture testing."""


class SimpleService:
    """A simple service class."""

    def __init__(self, name):
        self.name = name

    def get_name(self):
        return self.name

    def set_name(self, name):
        self.name = name


def standalone_function(x):
    return x * 2
